include/service/entities/content/article_renderer.js

```diff
@@ -1,42 +1,44 @@
 /*
-	Copyright (C) 2015  PencilBlue, LLC
+ Copyright (C) 2015  PencilBlue, LLC
 
-	This program is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation, either version 3 of the License, or
-	(at your option) any later version.
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
 
-	This program is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
 
-	You should have received a copy of the GNU General Public License
-	along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
 
 //dependencies
 var util  = require('../../../util.js');
 var async = require('async');
 
 module.exports = function(pb) {
-    
+
     //pb dependencies
     var DAO            = pb.DAO;
-    
+
     /**
-     * Retrieves the necessary data as well as prepares the layout so a view 
+     * Retrieves the necessary data as well as prepares the layout so a view
      * loader can complete the render of content
      * @class ArticleRenderer
      * @constructor
      */
-    function ArticleRenderer(context) {
+    function ArticleRenderer(context, options) {
         if (context) {
             this.commentService = new pb.CommentService(context);
             this.hostname = context.hostname;
         }
+        this.site = pb.SiteService.getCurrentSite(options.site);
+        this.onlyThisSite = options.onlyThisSite;
     }
```

Comment:
There isn't a need for an options object here.  Just provide the property as part of the context object.  They are synonymous.